doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,301 @@
+      set verbosity             = verbose
+      set calculate force       = true
+      set calculate torque      = false
+      set force name            = force
+      set output precision      = 10
+      set calculation frequency = 1
+      set output frequency      = 1
+    end
+    
+Mesh 
+~~~~~~~~~~~~~~~~~~
+
+A C-Type mesh was created around the NACA. It is the usual type of mesh chosen for airfoils because it allows for the curvature of the grid to match the leading edge of the airfoil. Also, it allows the user to place more cells on areas which require higher resolution, typicaly the upper and lower part of the airfoil. In order to obtain such a mesh, the Transfinite functions of ``gmsh`` were used. To get a good understanding of how it was done, the reader is advised to read through the ``cTypeMesh.geo`` file included in the example which is thoroughly commented. To generate a mesh with a different angle, the only thing required is to change the ``angle`` parameter in the ``cTypeMesh.geo`` file.
+
+.. code-block:: text
+	
+    subsection mesh
+      set type      = gmsh
+      set file name = naca.msh
+    end
+
+Below is the whole mesh and a zoom on the airfoil, for an angle of attack :math:`\alpha = 15°`
+
+.. image:: image/wholeMesh.png
+
+.. image:: image/zoomedMesh15.png
+
+Mesh adaptation 
+~~~~~~~~~~~~~~~~~~
+
+Mesh adaptation is used to get a higher resolution in areas of interest, that is to say close to the airfoil, while keeping a coarse mesh far from the NACA. Since the mesh is big and the simulation lengthy in time, it was chosen not to refine too much. The parameters were tuned as follow : 
+
+.. code-block:: text
+	
+    subsection mesh adaptation
+      set type                 = kelly
+      set variable             = velocity
+      set fraction type        = number
+      set max number elements  = 700000
+      set max refinement level = 2
+      set min refinement level = 0
+      set frequency            = 5
+      set fraction refinement  = 0.02
+      set fraction coarsening  = 0.16
+    end
+    
+    
+FEM
+~~~
+
+In this example, the interpolation order has been set to one for both velocity and pressure.
+
+.. code-block:: text
+
+    subsection FEM
+      set pressure order = 1
+      set velocity order = 1
+    end
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+The boundary conditions are defined as presented above : 
+
+.. code-block:: text
+
+     subsection boundary conditions
+       set number = 4
+       subsection bc 0
+         set type = noslip
+       end
+       subsection bc 1
+         set type = function
+         subsection u
+           set Function expression = 1
+         end
+         subsection v
+           set Function expression = 0
+         end
+         subsection w
+           set Function expression = 0
+         end
+       end
+       subsection bc 2
+         set type = slip
+       end
+       subsection bc 3
+       	set type = outlet
+       	set beta = 1.3
```

Comment:
1.3 is a very odd value, why 1.3 and not like 10 or 100?

Reply:
Since it is an outlet condition, "beta should be close to unity" according to the Lethe doc on boundary conditions, so I thought 1.3 would be fitting and it proved to be working afterwards, so I decided not to change it.

Reply:
Ok, it's a bit random but no biggy.